_translations/ko.yml

```diff
@@ -118,6 +118,7 @@ ko:
     wallettrezor: "TREZOR는 편리함을 희생하지 않고 높은 수준의 보안을 제공하는 하드웨어 지갑입니다. 콜드 스토리지와 달리, TREZOR는 온라인 장치에 연결된 상태에서 거래에 서명할 수 있습니다. 즉, 비트코인을 사용하는 것은 위험에 노출된 컴퓨터를 사용할 시에도 안전하다는 것을 의미합니다."
     walletnano: "Ledger Nano는 하드웨어 지갑이며 ST23YT66 은행 스마트카드 플랫폼으로 만들어졌습니다. 사용자 개인키를 안전하게 보호하고, 보안 선불 카드 또는 멀티 서명으로 사용되는 거래를 확인합니다. 오픈 소스가 아닌 경우, 엄격하게 확인할 수 있습니다."
     walletwasabi: "와사비는 개인정보 보호를 중요시 여기는 비트코인 지갑입니다."
+    walletunstoppable: "월렛 뱅크는 비트코인, 이더리움, 비트코인 케쉬, 이알씨 20 토크들들을 포함한 주요한 암호통화를 지킬 안전하고 완전하게 분산화된 비밀 지갑 어플입니다. 지원되는 통화들의 수는 점점 추가되어 배포할 것입니다."
```

Comment:
Please ensure that the description is consistent with the English description.

Reply:
done